homeassistant/components/yamaha/media_player.py

```diff
@@ -161,7 +161,7 @@ async def async_setup_platform(
 
     entities = []
     for receiver in receivers:
-        if receiver.zone in config_info.zone_ignore:
+        if config_info.zone_ignore and receiver.zone in config_info.zone_ignore:
```

Comment:
I used this to fix the issue in mypy. Maybe there is a better way?

Reply:
This is actually an accurate fix IMHO
What makes you doubt it?

Reply:
I wasn't sure if we could move it out of the loop - but I couldn't see an easy/clean way to do it.